openquake/server/templates/engine/index.html

```diff
@@ -60,6 +60,25 @@
               <a href="#" data-calc-id="<%= calc.get('id') %>" is-running="<%= calc.get('is_running') %>" class="btn btn-sm btn-log" style="margin: 2px 0 2px 0;">Console</a>
               <% if (calc.get('is_running') == false) { %>
                 <a href="#" data-calc-id="<%= calc.get('id') %>" class="btn btn-sm btn-danger" style="margin: 2px 0 2px 0;">Remove</a>
+                <!-- confirm_modal -->
+                <div class="modal-backdrop  in hide back_confirmDialog<%= calc.get('id') %>"></div>
+                <div id="confirmDialog<%= calc.get('id') %>" class="modal hide" data-keyboard="true" tabindex="-1">
+                <div class="modal-dialog">
+                  <div class="modal-content">
+                    <div class="modal-header">
+                      <h4 class="modal-title">Removing Calculation</h4>
```

Comment:
I would use `s/Calculation/calculation/g` (lower `c`) for consistency with the modals.